translate/lang/data.py

```diff
@@ -161,7 +161,7 @@
     'se': (u'Northern Sami', 2, '(n != 1)'),
     'si': (u'Sinhala; Sinhalese', 2, '(n != 1)'),
     'sk': (u'Slovak', 3, '(n==1) ? 0 : (n>=2 && n<=4) ? 1 : 2'),
-    'sl': (u'Slovenian', 4, '(n%100==1 ? 1 : n%100==2 ? 2 : n%100==3 || n%100==4 ? 3 : 0)'),
+    'sl': (u'Slovenian', 4, 'n%100==1 ? 0 : n%100==2 ? 1 : n%100==3 || n%100==4 ? 2 : 3'),
```

Comment:
Sadly there is no consistency. Personally I prefer to use them.

But the true reason I raised it in this case is so the PR diff would just reflect the actual changes. I prefer to have cleanups and actual changes on separate commits.